### ISSUE TITLE
Missing a hyperparameters in stage1training

### ISSUE THREAD

**Issue by nutfly** [fjchange/MIST_VAD#10]
kl=F.binary_cross_entropy(outputs_mean,labs,reduction='mean')
loss=args.lambda_kl*kl+loss

lambda_kl is missing in train_MIL_Generator.py, I can not find this hyperparameters in the paper too. Can you give the specific value of lambda_kl？

**Comment by fjchange** [comment]
It is not useful. I have deleted it to avoid misunderstanding.
